.gitignore

```diff
@@ -22,6 +22,7 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+venv/
 
```

Comment:
@parthasarathi9438 Add migrations folder to .gitignore file and remove all the migration files. We can do the migration manually on the server

Reply:
added migrations file in gitignore kindly remove it